doc/mgr/ssh.rst

```diff
@@ -41,5 +45,90 @@ To clear this value use the command:
 Development
 -----------
 
-Instructions for setting up a development environment can be found in the Ceph
-source tree at `src/pybind/mgr/ssh/README.md`.
+
+There are multiple ways to set up a development environment for the SSH orchestrator.
```

Comment:
I'd be :+1: for completely removing the `Development` section form this documentation and instead put the content into `src/pybind/mgr/ssh/README.md` or `src/pybind/mgr/ssh/HACKING.rst`.

There are two locations for having documentation aimed at developers: 

* `/ceph/doc/dev` for regular developer documentation
* `**/HACKING.rst` (like in `/ceph/src/pybind/mgr/dashboard`) for component specific developer documentation. 

cc @LenzGr

Reply:
Right, I'd be in favor of the moving it to a separate page since it doesn't _really_ belong here.

`HACKING.rst` sounds fine to me